.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+- Please ensure code changes are split into a series of logically independent commits.
+- Every commit should have a subject/title (What) and a description/body (Why).
+- Every PR must have a description.
+- As an example you can use git commit -m"What" -m"Why" to achieve the requirements above. GitHub automatically recognises the commit description (-m"Why") in single commit PRs and adds it as the PR description.
+- Use the imperative mood in the subject line for every commit. E.g `Mark infraID as required` instead of `This patch marks infraID as required` (This follows Git’s own built-in conventions). See https://github.com/openshift/hypershift/pull/485 as an example.
+- See docs/content/contribute for more details.
```

Comment:
How about linking to the published docs (https://hypershift-docs.netlify.app/contribute/) and making having reviewed that a check list item?

Reply:
Updated the link.
I'd rather leave here as a general guideline you should probably read once, but keep the checklist for action items that apply to every single PR you create.